pkg/consts/consts.go

```diff
@@ -26,6 +26,7 @@ const (
 	LabelKeyCreatedBy        = "createdBy"
 	LabelValueKanister       = "kanister"
 	LabelPrefix              = "kanister.io/"
+	JobIDLabelSuffix         = "JobID"
```

Comment:
```suggestion
	LabelSuffixJobID         = "JobID"
```
Just to be consistent with existing const format

Reply:
Done